File: sample/src/main/java/com/example/java/CaseLineSpacing.java
package com.example.java;

public class CaseLineSpacing {
    public CaseLineSpacing(int foo, Runnable bar, Runnable baz) {
        switch (foo) {
            // Lorem ipsum.
            case 0:
                bar.run();

            /* Lorem ipsum. */
            case 1:
                bar.run();
                break;

            /* Lorem ipsum. */
            default:
                baz.run();
                break;
        }
    }
}

File: sample/src/main/kotlin/com/example/kotlin/CaseLineSpacing.kt
package com.example.kotlin

class CaseLineSpacing {
    fun foo() {
        when (0) {
            // Lorem ipsum.
            0 -> println()

            /*
             * Lorem ipsum.
             */
            1 -> {
                println()
            }

            /** Lorem ipsum. */
            else -> {
                println()
            }
        }
    }
}
